src/components/entity/FAQ.tsx

```diff
@@ -40,10 +41,13 @@ const FAQ = (props: FAQItem) => {
             size="1.5em"
           />
         </div>
-      </div>
+      </button>
       <div
         ref={faqRef}
-        className="overflow-y-hidden duration-500 h-0 transition-[height]"
+        className={classNames(
+          "overflow-hidden duration-500 h-0 transition-all",
+          { invisible: !isOpen }
```

Comment:
In my JSFiddle example, having transition-property include `visibility` only when collapsed was a key part of it. If the transition includes visibility when not collapsed, it means that the content won't become visible until the opening transition is finished instead of being revealed as it opens. I don't see how to do it in tailwind, but is there a way you can remove transition-all and apply `transition-property: height;` when not collapsed and `transition-property: height, visibility;` when collapsed?

Reply:
@bhainesva Never mind this. As discussed via slack, this is not necessary. It works fine when visibility is always a transition property: https://jsfiddle.net/L5fvm6b7/